types/amphtml-validator/validator.proto.d.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * Enums from protobufs
+ * https://github.com/ampproject/amphtml/blob/master/validator/validator.proto
+ */
+
+export type ValidationResultStatus =
+    | 'UNKNOWN'
+    | 'PASS'
+    | 'FAIL'
+
+export type ValidationErrorSeverity =
+    | 'UNKNOWN_SEVERITY'
+    // A document with at least one error of this severity fails validation.
```

Comment:
I think it's more common to put the comment above the thing it comments? It's obvious what you're doing here, less so if you're in the middle of the long list of `ErrorCategoryCode`.

Reply:
These were copied verbatim from the original codebase (a protobuf file). I've just removed all the comments.